package cloud.stock.auth.domain.exceptions;

public class UserDuplicatedException extends RuntimeException {
}
